pkg/utils/utils.go

```diff
@@ -357,30 +357,30 @@ func SetProxyEnvVars(proxyConfig *configv1.Proxy) []v1.EnvVar {
 			Name:  "HTTPS_PROXY",
 			Value: proxyConfig.Status.HTTPSProxy,
 		},
-		v1.EnvVar{
-			Name:  "https_proxy",
-			Value: proxyConfig.Status.HTTPSProxy,
-		})
+			v1.EnvVar{
```

Comment:
did `gofmt` do this?

Reply:
@ewolinetz yes, this is done by `gofmt`, `Makefile:94`

Reply:
The formatting does not look correct